README.md

```diff
@@ -13,207 +13,111 @@ Istanbul's state of the art command line interface, with support for:
 * applications that spawn subprocesses.
 * ES2015 transforms, via [`babel-plugin-istanbul`], or source-maps.
 
-## Instrumenting your code
+## Installation & Usage
 
-You can install nyc as a development dependency and add it to the test stanza
-in your package.json.
-
-```shell
-npm i nyc --save-dev
-```
+You can use `npx` instead of installing nyc as a dependency.
+Simply add `nyc` before your test runner is invoked (replace `mocha` with your test runner everywhere you see it):
 
 ```json
 {
   "scripts": {
-    "test": "nyc mocha"
+    "test": "npx nyc mocha"
   }
 }
 ```
 
-Alternatively, you can install nyc globally and use it to execute `npm test`:
+Or use your package manager to add a dev dependency: `npm i -D nyc` or `yarn add -D nyc`.
```

Comment:
I think I prefer the npx first approach because it gets rid of the problem of people having outdated versions of nyc, and as a bonus for those who prefer to have less in their project node_modules but still shares nyc across their machine. It's basically what `npm i -g` always wanted to be imo

I'll make sure to mention not having nyc already in your test script... and to make adjustments if you have to run other executables. Though with gulp specifically I'm not sure why that wouldn't be in your dependencies, you usually have to write a gulpfile based on the node api. Still a valid point, I'll try to figure out the wording today

Reply:
@bcoe I think (/feel strongly) the npx first is still more effective, and we can still have people install nyc@next with npx or `npm i`

I was debating whether I should specify `npx nyc@latest` in the default instructions, and I think I will so that future troubleshooting instructions can be to try `npx nyc@next` or whatever tag we need to specify. 

Finding out which version is not difficult, so we can add that to the issue template.

Reply:
@bcoe after chatting with @coreyfarrell I have been convinced. I have fixed this section so I'm resolving this.